### DISCUSSION TITLE
Can I see react Document  page  source code ??

### DISCUSSION THREAD

**Discussion by pupss89345** [UI5/webcomponents-react#6636] [Q&A] [answered]
Like title said It's possible see blue square page source code ?
![2024-11-18_17-24](https://github.com/user-attachments/assets/a93ed1b8-1240-4d6b-b404-b3a31fe92db6)

**Comment by Lukas742** [comment, marked as answer]
Hi @pupss89345 
we only show the column configuration for this example inside `Docs` as the rest is not really relevant:
https://sap.github.io/ui5-webcomponents-react/v2/?path=/docs/data-display-analyticaltable--docs#responsively-display-columns-on-small-devices-pop-in
